xbmc/games/addons/GameClient.cpp

```diff
@@ -713,6 +717,58 @@ bool CGameClient::Deserialize(const uint8_t* data, size_t size)
   return bSuccess;
 }
 
+void CGameClient::LoadSRAM()
+{
+  std::string path = URIUtils::AddFileToFolder(CProfilesManager::GetInstance().GetSavestatesFolder(), "SRAM");
+              path = URIUtils::AddFileToFolder(path, URIUtils::GetFileName(m_gamePath));
+
+  uint8_t *data;
+  size_t size;
+  m_pStruct->GetMemory(GAME_MEMORY_SAVE_RAM, &data, &size);
+
+  std::string sramPath = path + ".sram";
```

Comment:
Others use .srm or .sav. If we stick to .sav it's generic enough also for non-emulated games.